specificatie/adressering.yaml

```diff
@@ -31,9 +31,9 @@ components:
       type: string
       maxLength: 40
       description: |
-        Het eerste deel van een adres is een combinatie van de straat en huisnummer.
+        Het eerste deel van een adres is een combinatie van de korteStraatnaam en huisnummer.
```

Comment:
is dat hier nodig? Voor iemand die adresregels wil gebruiken is het niet duidelijk wat je bedoelt met korteStraatnaam, terwijl straat of straatnaam wel direct duidelijk is. Zou jammer zijn dat de gebruiker vervolgens alsnog in verblijfadres moet gaan opzoeken wat bedoeld wordt met korteStraatnaam.
Je hoeft hier niet exact uit te leggen hoe de adresregel wordt samengesteld, alleen wat het is. En wat het is is de straatnaam en huisnummer.

Reply:
Aangepast